health/health.d/net.conf

```diff
@@ -1,6 +1,24 @@
 
 # you can disable an alarm notification by setting the 'to' line to: silent
 
+# -----------------------------------------------------------------------------
+# net traffic overflow
+
+ template: 10s_net_traffic_overflow
+       on: net.net
+       os: linux
+    hosts: *
+ families: *
+   lookup: average -10s unaligned absolute of sent,received
+     calc: $this * 100 / ($nic_speed_max * 1000)
+    units: %
+    every: 10s
+     warn: $this > (($status >= $WARNING)  ? (50) : (70))
+     crit: $this > (($status == $CRITICAL) ? (70) : (90))
```

Comment:
These conditions combined with the `10s` duration of the `lookup` lines, will make the alarm quite chatty (i.e. it will send alarms frequently).

I suggest to change the `lookup` line to `-1m` and set warning `> 90` and critical to `>95`.

If you do this, also change the title to `1m_XXX`.

Reply:
updated duration to 1m, but I made the threshold a little bit lower, since it's kind of rare(and hard) to keep a constant network load at near full speed for the whole minute.